packages/scapi/README.md

```diff
@@ -0,0 +1,26 @@
+# `@supercollider.js/scapi`
```

Comment:
nice, is this new proposed tcp solution?

Reply:
No, that's a different system.  https://github.com/crucialfelix/API

It's nice to have it separated now into its own package (and there will be more to come). People who just want to run sclang with a node interface can do so without installing everything else.

Reply:
wow, i'm impressed by the amount of work behind this. nice